src/Fixer/FunctionNotation/FopenFlagsFixer.php

```diff
@@ -21,21 +24,42 @@
 /**
  * @author SpacePossum
  */
-final class FopenFlagsFixer extends AbstractFopenFlagFixer
+final class FopenFlagsFixer extends AbstractFopenFlagFixer implements ConfigurationDefinitionFixerInterface
 {
+    /**
+     * {@inheritdoc}
+     */
+    public function configure(array $configuration = null)
+    {
+        parent::configure($configuration);
+    }
+
     /**
      * {@inheritdoc}
      */
     public function getDefinition()
     {
         return new FixerDefinition(
-            'The flags in `fopen` calls must contain `b` and must omit `t`.',
+            'The flags in `fopen` calls must omit `t` and `b` must be used consistent.',
```

Comment:
I think this would read better:

```suggestion
            'The flags in `fopen` calls must omit `t`, and `b` must be omitted or included consistently.',
```

Reply:
Im following the [SF doc standard](https://symfony.com/doc/current/contributing/documentation/standards.html) ; 
`Punctuation: avoid the use of Serial (Oxford) Commas;`

Reply:
Reading it left-to-right reads like this to me:

> The flags in `fopen` calls must omit `t` and `b`

then you get to

> must be...

and you have to backtrack to understand the sentence fully. Also, I believe an Oxford Comma is only considered that when listing a series of three or more things.

Reply:
:+1:  merged, can you fix the readme one as well?